tests/subsys/dfu/mcuboot/src/main.c

```diff
@@ -26,7 +26,7 @@ void test_bank_erase(void)
 	flash_dev = device_get_binding(DT_FLASH_DEV_NAME);
 
 	for (offs = FLASH_AREA_IMAGE_1_OFFSET;
-	     offs <= FLASH_AREA_IMAGE_1_OFFSET + FLASH_AREA_IMAGE_1_SIZE;
+	     offs < FLASH_AREA_IMAGE_1_OFFSET + FLASH_AREA_IMAGE_1_SIZE;
```

Comment:
This file changes do not seem to belong in this commit..

Reply:
WIll move this to a seperate PR

Reply:
Moved to PR #12599 , will rebase this PR once merged